### ISSUE TITLE
Animated Java Blueprint

### ISSUE THREAD

**Issue by LeGriffom** [Provim-Gaming/nylon#6]
Hey,

Will the mod be able to use AJ blueprints in the future? (new model format for Animated Java projects: https://animated-java.dev/docs/the-blueprint-format/overview)

Thank you,

**Comment by Wesley1808** [comment]
Support for the new AnimatedJava versions are being worked on (see https://github.com/Provim-Gaming/nylon/tree/feature/animatedjava-1.0.0).

It is mostly for the plugin mode json exports though, not the blueprint file you open with blockbench itself (similar to before).
In 1.0.0+ there is no longer need for a separate resourcepack to be exported though, which makes things a lot easier - they're part of the json export now, allowing nylon to automatically generate the resourcepack for it through polymer.

**Comment by LeGriffom** [comment]
Yeah I saw the few differences between the two export methods after updating my models. It's great that you're already working on it. Good luck, and your mod is fire by the way! Congrats
